docs/book/features/block_volume.md

```diff
@@ -250,6 +250,8 @@ There are many ways to create static PV and PVC binding. Example: Label matching
 
 Static Volume Provisioning is supported only in Vanilla Kubernetes clusters but not in Supervisor clusters
 
+**NOTE:** For Block volumes, vSphere Cloud Native Storage (CNS) only allows one PV in the Kubernetes cluster to refer to a storage device.
```

Comment:
"refer to a storage device" => "refer to a storage disk"

Reply:
Can you mention, Creating multiple PV using the same block volume handle is not supported.